source/help/faq/whytex.md

```diff
@@ -1,23 +1,23 @@
+We mean by this the relationship between equations and their labels, references and their numbers, subsection headings and their entries in the table of contents, and so on. While ordinarily available in TeX/LaTex source, conversion to Postscript irretrievably loses this structural information. The loss is unfortunate because, with new formats such as PDF, the information can be used to provide active hyperlinks: e.g., in a PDF viewer, you can click on an equation number and jump back to the specified equation. Moreover, TeX itself can be processed as HyperTeX, and with the proper dvi previewer, clicking on equation numbers will bring up the desired equation in a separate window or even retrieve other papers specified by their proper archive/papernum identifier. HyperTeX works by redefining the standard macros and works retroactively for pre-existing TeX/LaTeX sources -- HyperTeX conversion is accomplished by merely re-TeXing with the modified macros. Since information is ordinarily lost in each processing stage, the TeX source contains (close to) the maximal amount of contextual information that can be retroactively processed into any future format that can take advantage of it. For more information, see [hypertex help](../../help/hypertex/index.md).
 
 
 Why doesn't arXiv Accept Preprocessed Submissions?
 -------------------------------------------------------------
 
-  *  arXiv cannot easily process pre-processed documents into more accessible formats such as HTML. 
-  *  Source files are of higher archival value than converted documents because they can provide a foundation upon which we can develop future services and applications to maintain access and readability of papers indefinitely.
+  *  arXiv cannot easily process pre-processed documents into more accessible formats like HTML. 
```

Comment:
Hm, actually [such as](https://www.grammarly.com/blog/grammar/like-vs-such-as/) was already the better choice here.